keps/sig-architecture/3136-beta-apis-off-by-default/README.md

```diff
@@ -0,0 +1,311 @@
+existing discovery mechanisms (example: kubectl's api-resources sub command or the `/api/apps/v1` REST API).
+
+### Risks and Mitigations
+
+Adoption of beta features will slow.
+Given how kubernetes is now treated, this is a good thing, not a bad thing.
+Those users that want to move quickly and get new features can do so by enabling all beta feature
+or just enabling those that are important for their workload.
+The [PRR survey](https://datastudio.google.com/reporting/2e9c7439-202b-48a9-8c57-4459e0d69c8d/page/Cv5HB) shows that 
+over 30% of cluster-admins have enabled alpha features on at least some production clusters, so cluster-admins are willing and able to enable features
+that are not on by default when they are desired.
+
+If two or more APIs are tightly coupled together, it will now be possible to enable them independently.
+This can lead to unanticipated failure modes, but should only impact beta APIs with beta dependencies.
+While this is a risk, it is not very common and components should fail safe as a general principle.
+
+If beta APIs are off by default, it's possible that fewer clients will use them and provide feedback.
+This is a risk, but early adopters are able to enable these features and have a history of enabling alpha features.
+When moving from beta to GA, it will be important for sigs to explicitly seek feedback.
+We will address this by extending the PRR questionnaire to include a GA-targeted question to validate that the feature
+was reasonably validated in production use-cases.
+
+If beta APIs are off by default, it is possible that sigs don't treat taking an API as an indication of a "mostly-baked" API.
+If this happens, then more transformation may be required.
+Keeping our beta API rules consistent and continuing to enforce easy to use APIs seems to be the best option.
+
+## Design Details
+
+<!--
+This section should contain enough information that the specifics of your
+change are understandable. This may include API specs (though not always
+required) or even code snippets. If there's any ambiguity about HOW your
+proposal will be implemented, this is the place to discuss them.
+-->
+
+### Test Plan
+
+Integration tests will be written to ensure that no new beta APIs are enabled in the kube-apiserver by default.
+Unit tests will be written to ensure that the new flag functionality works as expected.
+
+### Graduation Criteria
+
+This KEP is a policy KEP, not a feature KEP.  It will start as GA.
+
+#### GA
+
+- Integration and unit tests from above.
+- updating the enablement docs for beta
+  - https://kubernetes.io/docs/reference/using-api/#api-versioning
+  - https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#using-a-feature 
+    Even though that is talking about feature gates, it is likely worth calling out there that new beta REST APIs are no
+    longer enabled by default)
```

Comment:
I also recommend revising / expanding the [feature state shortcode](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-state) with a parameter to indicate whether or not the feature is enabled by default.